src/main/proto/wfa/measurement/api/v2alpha/measurement_spec.proto

```diff
@@ -0,0 +1,49 @@
+// Copyright 2021 The Cross-Media Measurement Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package wfa.measurement.api.v2alpha;
+
+import "wfa/measurement/api/v2alpha/differential_privacy.proto";
+
+option java_package = "org.wfanet.measurement.api.v2alpha";
+option java_multiple_files = true;
+option java_outer_classname = "MeasurementSpecProto";
+
+// Specification for a `Measurement`. Immutable.
+message MeasurementSpec {
+  // Encryption public key for the `Measurement` that this `MeasurementSpec` is
+  // associated with. Required.
+  bytes measurement_public_key = 1;
+
+  message ReachAndFrequency {
+    // Differential privacy parameters for the global reach DP noise registers.
+    //
+    // Each of these registers contains: a random register ID which is outside
+    // of the bounds of the normal ID space, the same constant key indicating
+    // that the register is destroyed, and an arbitrary count value.
```

Comment:
I am not sure if such details are appropriate in the api documentation.

@kungfucraig how should we describe these params to MC?